app/utils.js

```diff
@@ -34,10 +34,26 @@ export function stringDurationToSeconds (duration) {
   return 0;
 }
 
-export function getSelectedStream (streams, defaultMusicSource) {
+export function getSelectedStream(streams, defaultMusicSource) {
   let selectedStream = _.find(streams, { source: defaultMusicSource });
 
   return typeof selectedStream === 'undefined'
     ? streams ? streams[0] : null
     : selectedStream;
 }
+
+export function removeQuotes(text) {
+  return String(text).replace(/[“”]/g, '');
+}
+
+export function createLastFMLink(artist, track) {
```

Comment:
Should be tested with at least few examples of tricky artist names so we're sure it'll cover most of the typical cases

Reply:
Add 2 more advanced cases to tests